src/BottomNavigation/BottomNavigation.js

```diff
@@ -0,0 +1,56 @@
+// @flow weak
+
+import React, { PropTypes, Children, cloneElement } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('BottomNavigation', (theme) => {
+  return {
+    root: {
+      position: 'relative',
+      width: '100%',
```

Comment:
What about removing this property?
We would need to add an extra `div` wrapper in the demo with a `flexGrow: 1` property to make sure we use all the available space.
I think that it's better to let the parent handle the layout.